recipes/hdf4/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib
+fi
+
+export CXXFLAGS="${CFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export CFLAGS="${CFLAGS} -pipe -O2 -fPIC -I${PREFIX}/include"
+
+chmod +x configure
+
+# The --enable-silent-rules is needed because Travis CI dies on the long output from this build.
+./configure \
+    --disable-static \
+    --enable-linux-lfs \
+    --with-ssl \
+    --with-zlib \
+    --with-jpeg \
+    --disable-netcdf \
```

Comment:
I am not an expert, but I don't think we want `netcdf 4` support here. (I am not even sure it would work with `netcdf 4`!)

However, we do want the opposite: `hdf4` support in our `netcdf 4` package. (AKA `libnetcdf`)